benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -433,3 +359,90 @@ msgid "enrollment.pages.success.p2"
 msgstr ""
 "When boarding transit, pay with this same card, and your discount will "
 "automatically be applied to your fare purchase."
+
+msgid "eligibility.pages.index.dmv.label"
+msgstr "Senior Discount Program"
+
+msgid "eligibility.pages.index.mst.label"
+msgstr "MST Courtesy Cardholder"
+
+msgid "eligibility.pages.confirm.dmv.content_title"
+msgstr "Let’s see if we can confirm your age with the DMV"
+
+msgid "eligibility.pages.start.dmv.items[0].title"
+msgstr "Your California ID"
+
+msgid "eligibility.pages.start.dmv.items[0].text"
+msgstr "Driver’s license or ID card"
+
+msgid "eligibility.pages.confirm.dmv.p[0]"
+msgstr ""
+"If you’re 65 or older, we can confirm you are eligible for a senior "
+"discount when you ride transit. We don’t save your information."
+
+msgid "eligibility.forms.confirm.dmv.fields.sub"
+msgstr "CA driver’s license or ID number"
+
+msgid "eligibility.forms.confirm.dmv.fields.name"
+msgstr "Last name (as it appears on ID)"
+
+msgid "eligibility.pages.start.dmv.content_title"
+msgstr "Great, you’ll need two things before we get started..."
+
+msgid "eligibility.pages.start.dmv.p[0]"
+msgstr ""
+"This program is currently open to those who are 65 or older. Not over 65? "
+"Get in touch with your transit provider to learn about other available "
+"discount programs."
+
+msgid "eligibility.pages.confirm.dmv.title"
+msgstr "Confirm your age"
+
+msgid "eligibility.pages.unverified.dmv.p[0]"
+msgstr ""
+"You may still be eligible for a discount, but we can’t verify your age "
+"with the DMV."
+
+msgid "eligibility.forms.confirm.mst.fields.sub"
+msgstr "MST Courtesy Card number"
+
+msgid "eligibility.forms.confirm.mst.fields.name"
+msgstr "Last name (as it appears on Courtesy Card)"
+
+msgid "eligibility.pages.start.mst.content_title"
+msgstr "You’ll need two things before we get started:"
+
+msgid "eligibility.pages.start.mst.items[0].title"
+msgstr "Your MST Courtesy Card"
+
+msgid "eligibility.pages.start.mst.items[0].text"
+msgstr "An active card that has not expired"
+
+msgid "eligibility.pages.start.mst.p[0]"
+msgstr ""
+"This program is currently open to all MST Courtesy Cardholders. Not a "
+"Courtesy Cardholder? Get in touch with your transit provider to learn "
+"about other available discount programs at"
```

Comment:
**question**: Will a separate issue handle adding the transit agency phone number after this string?

Reply:
I created #349